.travis.yml

```diff
@@ -15,6 +15,8 @@ matrix:
         - python: 3.8
           env: TOXENV=qa,doc
 
-install: pip install --ignore-installed --upgrade setuptools pip tox coveralls
+install:
+  - pip uninstall virtualenv
```

Comment:
see: https://travis-ci.org/github/rm-hull/luma.core/jobs/740002770#L202-L207

Add a `-y` flag maybe ?

Reply:
ill rework this pr this weekend. that last commit was a (dumb) test.